app/src/main/res/layout/activity_benchmark_mode.xml

```diff
@@ -25,10 +25,11 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:layout_marginStart="104dp"
-      android:layout_marginTop="104dp"
+      android:layout_marginTop="220dp"
       android:layout_marginEnd="104dp"
       android:text="@string/BenchmarkMode_MakePhoto_Text"
       app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintHorizontal_bias="0.0"
```

Comment:
Change to 0dp just to make things unified

Reply:
+1